Intersect.Server.Core/Entities/Entity.cs

```diff
@@ -2084,6 +2084,10 @@ public virtual void TryAttack(Entity target)
             if (this is Player && !(enemy is Resource))
             {
                 var lifestealRate = thisPlayer.GetEquipmentBonusEffect(ItemEffect.Lifesteal) / 100f;
+                    if (lifestealRate < 0)
```

Comment:
remove this if, regardless of the value being negative, it will not pass the next check on the line 2094

Reply:
I don't understand why he shouldn't pass the next inspection? After all, this is just an exception which, if there is a minus value, sets it to 0, i.e. the default basic amount. This change must be made, otherwise lifesteal will not work properly, as I have already explained. unless I don't understand what you mean

Reply:
unless we want the negative lifesteal and manasteal to take the user's life after the attack. then instead, you can give a condition for basedamage that if basedamage is a negative value, it changes the sign from - to +. This is due to the problem with healing spells, which in their recipe must have negative values ​​to heal.

Reply:
I will explain this further. If we didn't have this "if", then if we had -30% lifesteal and used a healing spell whose formula had e.g. -10 to heal by 10, then there would be an error in the code. because lifestealrate -30% * basedamage -10, gives a positive result, which causes additional treatment.